tests/test_api_endpoints.py

```diff
@@ -115,32 +117,40 @@ def test_system_groups(auth_token, db_setup, db_create_account, db_create_system
 
     system = db_get_record(System)
     system.groups = [{
-        "id": "fd11209a-1ca7-49b4-ae27-0f8a365b95b8",
+        "id": "12345678-fe1b-4191-8408-cbadbd47f7a3",
         "name": "ros-test-3"
     }]
     db.session.commit()
 
     with app.test_client() as client:
+        mock_enable_rbac(mocker)
+        mock_rbac(get_rbac_mock_file("mock_rbac_returns_groups_including_example_group.json"), mocker)
+
+        mock_unleash_hbi_flag_enabled(mocker)
```

Comment:
@upadhyeammit is it possible to add a test case when unleash flag is false and check for default values for groups in that case?